app/views/issues/_reports.html.erb

```diff
@@ -0,0 +1,19 @@
+<% reports.each do |report| %>
+  <div class="reports">
```

Comment:
As this is inside the loop, and hence only contains one report, this seems like the wrong class name to use?